spec/mailers/digest_mailer_spec.rb

```diff
@@ -21,7 +21,6 @@
 
     it "includes the tracking pixel" do
```

Comment:
I think we can remove this whole test, it has no expectations now

Reply:
Wow, how'd I miss that?! 😆